### PR TITLE
[math] feat: add minus operation

### PR DESCRIPTION
**Pull request by anlia260**
# strategy.math support (minus)

Changes proposed in this pull request:

-  update example.json address formatter to pass test
-  add Example simple contract call  minus

### CODE REVIEW

File: src/strategies/math/README.md [hunk 1]
```diff
@@ -17,6 +17,8 @@ Currently supported operations are:
 | `a-if-lte-b`  | 3             | (x, a, b) = x <= b ? a : x                 |
 | `a-if-gt-b`   | 3             | (x, a, b) = x > b ? a : x                  |
 | `a-if-gte-b`  | 3             | (x, a, b) = x >= b ? a : x                 |
+| `sum`         | 2             | x + a                                      |
```
Comment (ChaituVR) [inline src/strategies/math/README.md:20]: Hi @anlia260 But using two strategies in a space will already return a sum, is there a reason why we need this?

**Review by ChaituVR — CHANGES_REQUESTED** [review]

File: src/strategies/math/README.md (hunk 1 above)
Comment (anlia260) [reply src/strategies/math/README.md:20]: sure, there is some reasons
1. using two strategies in a space will already return a sum is not too obvious ( detailed descriptions cannot be found in the documentation)
2. multiple strategies is less than eight, the math.sum is reduce sum that means multiple strategies expand in the feature ( but now just two)

Comment (anlia260) [reply src/strategies/math/README.md:20]: Got you worried, I will remove the sum now, waiting for me.

File: src/strategies/math/examples.json [hunk 2]
```diff
@@ -1,4 +1,106 @@
...
+                  "constant": true,
+                  "stateMutability": "view"
+                }
+              }
+            }
+          }
+        ],
+        "operation": "minus"
+      }
+    },
```
Comment (ChaituVR) [inline src/strategies/math/examples.json:95]: Can add smaller examples plz 😄 maybe like ticket strategy or something, Everything else looks fine, thanks for the updates

**Review by ChaituVR — CHANGES_REQUESTED** [review]

**Comment by ChaituVR** [comment]
Thanks for understanding and modifying it accordingly 🙏 :)

**Pull request merged**